tests/test_minhash.py

```diff
@@ -559,6 +559,23 @@ def test_similarity_1(track_abundance):
     assert round(b.similarity(b), 3) == 1.0
 
 
+def test_copy(track_abundance):
+    a = MinHash(20, 21, track_abundance=track_abundance)
+    a.add_hash(5)
+    a.add_hash(6)
+    b = a.copy()
+    assert a == b
+
+
+def test_frozen_copy(track_abundance):
+    a = MinHash(20, 21, track_abundance=track_abundance)
+    a.add_hash(5)
+    b = a.to_frozen()
```

Comment:
note this does not use `a.copy()`, and I'm guessing it probably should?

(I think we have tests for `to_frozen()` already.)

Reply:
Here, I was trying to check if the `to_frozen` is working and I realized that `to_frozen` also copies the minhash, so I removed the copy statement that I had. Should I put it back?

Reply:
hmm, not sure. What I'm thinking is that this test function doesn't actually test `copy()` in any way, and, I think, tests the same behavior as `test_frozen_and_mutable_2`.

If this is meant to be a test for `a.copy()` then you should replace `b = a.to_frozen()` with `b = a.copy()`, *even if* they are currently implemented the same way.

Reply:
okay!